api/homepage/radarr.php

```diff
@@ -349,17 +361,21 @@ public function formatRadarrCalendar($array, $number, $url)
 		$gotCalendar = array();
 		$i = 0;
 		foreach ($array as $child) {
-			if (isset($child['physicalRelease'])) {
+			if (isset($child['physicalRelease']) || $this->config['radarrCinemaRelease']) {
```

Comment:
I would change this to:
```
if (isset($child['physicalRelease']) || isset($child['inCinemas'])) {
```
This way we can test the array for both items then we can then add the check down below

Reply:
I can do that. I also want to check against those movies in that issue.